test/WebJobs.Script.Tests.Integration/ApplicationInsights/ApplicationInsightsEndToEndTestsBase.cs

```diff
@@ -80,15 +78,16 @@ public async Task Validate_Manual()
                 }
             }
 
+            // TODO: Re-enable this when we can override IMetricsLogger
             // App Insights logs first, so wait until this metric appears
-            string metricKey = MetricsEventManager.GetAggregateKey(MetricEventNames.FunctionUserLog, functionName);
-            IEnumerable<string> GetMetrics() => _fixture.MetricsLogger.LoggedEvents.Where(p => p == metricKey);
+            // string metricKey = MetricsEventManager.GetAggregateKey(MetricEventNames.FunctionUserLog, functionName);
```

Comment:
Are you working on an immediate follow up to reenable all this? Do we need a tracking issue so we don't forget this?

Reply:
I'm following up ASAP. Will consult with @fabiocav about the right way to handle this.